draft-ietf-quic-ack-frequency.md

```diff
@@ -411,18 +411,25 @@ controllers.
 ## Congestion Control
 
 A sender needs to be responsive to notifications of congestion, such as
-a packet loss or an ECN CE marking. Also, window-based congestion controllers
-that strictly adhere to packet conservation, such as the one defined in
-{{QUIC-RECOVERY}}, rely on receipt of acknowledgments to send additional data into
-the network, and will suffer degraded performance if acknowledgments are delayed
-excessively.
-
-To enable a sender to respond to potential network congestion, a sender SHOULD
-cause a receiver to send an acknowledgement at least once per RTT if there are
-unacknowledged ack-eliciting packets in flight. A sender can accomplish this by
-sending an IMMEDIATE_ACK frame once per round-trip time (RTT), or it can set the
-Ack-Eliciting Threshold and Request Max Ack Delay values to be no more than a
-congestion window and an estimated RTT, respectively.
+a packet loss or an ECN CE marking.  To enable a sender to respond to potential
+network congestion, a sender SHOULD cause a receiver to send an acknowledgement
+at least once per RTT if there are unacknowledged ack-eliciting packets in flight.
+A sender can accomplish this by sending an IMMEDIATE_ACK frame once per
+round-trip time (RTT), or it can set the Ack-Eliciting Threshold and
```

Comment:
Setting the Ack-Eliciting Threshold has no effect on whether the receiver will acknowledge.  It only works if that many packets are sent, which is something that I don't think you can guarantee when setting the value.

Reply:
I agree there is a bound on how long the transport should defer reacting when it has sent data - this is a control loop. For an Internet path, to me this is measured in RTTs, or part of an RTT. (I don't know whether we can state the same constraints to apply in both a heterogenous Internet and in a high speed DC). I was pretty sure that previous detailed analysis of feedback systems showed requirements for stable feedback at multiple control signals per RTT, so I'd encourage care here.

Reply:
he /or/ is still problematic - it's not one or the other, because the time is what actually matters for protocol robustness; and the count threshold is really helpful when there are path characteristic changes, because it prevents very large numbers of packets per ACK - or we simply wish to reduce the rate. Perhaps two sentences would be wiser.